article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -140,7 +144,10 @@ object ArticlePageChecks {
       "tone/performances",
       "tone/polls",
       "tone/profiles",
+      "tone/q-and-as",
+      "tone/quizzes",
```

Comment:
Quiz atoms which are not supported - maybe I ought to check older ones for these?